webpack.config.ts

```diff
@@ -0,0 +1,22 @@
+import path from 'node:path'
+import { Configuration } from 'webpack'
+
+const config: Configuration = {
+  entry: './src/client/index.ts',
+  context: path.resolve(__dirname),
+  mode: 'production',
+  devtool: 'source-map',
+  output: {
+    library: {
+      name: 'MODUK',
+      type: 'umd',
+    },
+    filename: 'moduk-frontend.umd.js',
+    path: path.resolve(__dirname, './dist/client'),
+  },
+  resolve: {
+    extensions: ['.ts', '.js', '.mjs'],
```

Comment:
`...` can be used to include the default list of extensions. So we may want to do something like:

```suggestion
    extensions: ['.ts', '.mjs', '...'],
```